ynput/ayon-core
# Resolve: version swap retain offset and IDT

## Changelog Description
Swapped version should retain the offset and IDT.

## Additional info
Ayon-core port of https://github.com/ynput/OpenPype/pull/6128 and 
closes https://github.com/ynput/OpenPype/issues/6125 issue

## Testing notes:
1. Load latest version of the clip that has Handles start > 0
2. Set any colorspace to media clip (this will need to have enabled colormanagement on Resolve project)
3. Use manage to swap for earlier version
4. Notice that colormanagement configuration is the same as the previous clip

Note that earlier version doesn't match the timeline trim of the latest version, and IDT of the version resets to default
